src/components/editableTable/columnsProps.js

```diff
@@ -16,7 +16,8 @@ const columnsProps = PropTypes.shape({
     clearSearch: PropTypes.func,
     closeRowOnClickAway: PropTypes.func,
     search: PropTypes.func,
-    selectSearchResult: PropTypes.func
+    selectSearchResult: PropTypes.func,
+    minimumSearchTermLength: PropTypes.number
```

Comment:
This is annoying but could this be added to the SingleEditTable and GroupEditTable stories mdx files? There's some markdown there outlining the columns props. 

That should hopefully be temporary, there's a storybook update coming that allows args tables to show nested properties but it can't just now